src/index.d.ts

```diff
@@ -0,0 +1,372 @@
+declare module '@mojaloop/central-services-shared' {
```

Comment:
shouldn't be here `declare namespace` like in this repo: https://github.com/mojaloop/sdk-standard-components/blob/master/src/index.d.ts
`declare module` should be used in ambient files

Reply:
thanks. done.